content/data-sharing/persistent-data-structures.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Persistent data structures"
+section: "Data Sharing"
+menu:
+  toc:
+    parent: "data-sharing"
+    weight: 20
+---
+
+## Problem
+
+You need to send mutable data from one actor to another while keeping a copy of it in your original actor.
+
+## Solution
+
+```pony
+use "collections/persistent"
+
+actor Collector
+  """
+  Receives characters via it's `collect` behavior and stores them.
+  Every 10 characters we receive results in the entire array being sent all to
+  the receiver.
+  """
+  let _receiver: Receiver
+  var _data: List[U8] = Lists[U8].empty()
```

Comment:
I would recommend using the persistent `Vec` in these examples. I think it would be best to guide people toward that, instead of `List`, since it's performance characteristics more closely resemble `Array` which you use in the other patterns.

Reply:
That's reasonable.